installer/algorand@.service

```diff
@@ -0,0 +1,38 @@
+## For some data directory $ALGORAND_DATA
+##
+## Setup:
+## ALGORAND_DATA=/var/lib/algorand_devnet
+## sudo mkdir -p ${ALGORAND_DATA}
+## sudo cp -p /var/lib/algorand/genesis/devnet/genesis.json ${ALGORAND_DATA}/genesis.json
+## sudo cp -p /var/lib/algorand/system.json ${ALGORAND_DATA}/system.json
+## sudo chown -R algorand ${ALGORAND_DATA}
+##
+## systemd allows for config overrides in a .conf file under a directory named for the service. e.g.:
+##
+## sudo mkdir -p /lib/systemd/system/algorand@$(systemd-escape ${ALGORAND_DATA}).service.d
+## cat <<EOF>/lib/systemd/system/algorand@$(systemd-escape ${ALGORAND_DATA}).service.d/override.conf
+## [Service]
+## User=someuser
+## EOF
+##
+## sudo systemctl enable algorand@$(systemd-escape ${ALGORAND_DATA})
+## sudo systemctl start algorand@$(systemd-escape ${ALGORAND_DATA})
+
+
+[Unit]
+Description=Algorand daemon for %I
+After=network.target
+
+[Service]
+ExecStart=/usr/bin/algod -d %I
+PIDFile=%I/algod.pid
+User=algorand
+Group=nogroup
+Restart=always
+RestartSec=5s
+ProtectSystem=full
+ProtectHome=true
+
+[Install]
+WantedBy=multi-user.target
+DefaultInstance=-var-lib-algorand
```

Comment:
It seems like we don't want two services (algorand and algorand@) fighting over /var/lib/algorand.  What's the reason for having a default instance here?

Reply:
I was hoping that a default instance would allow for `algorand@` or `algorand` to exist based on the `algorand@.service` but I couldn't get those to work. Could probably drop the default instance.